lib/store/index.js

```diff
@@ -34,6 +37,20 @@ class Store {
     return this.editor ? this.editor.getPath() : null;
   }
 
+  @computed
+  get notebook(): ?string {
+    const editor = this.editor;
+    let notebook = commutable.emptyNotebook;
+    const cellRanges = codeManager.getCells(editor);
+    _.forEach(cellRanges, cell => {
+      const { start, end } = cell;
+      const source = codeManager.getTextInRange(editor, start, end);
+      const newCell = commutable.emptyCodeCell.set("source", source);
```

Comment:
This makes me realize I was being a bit cute when I made the `emptyCodeCell` API. "Well, since it's immutable, we can just export the base object". This makes me think a wrapper function would look more clean here, especially if it took the initial source, etc., similar to the nbformat API.

Reply:
I think there was a ``createCodeCell`` somewhere inside, but it wasn't exposed.

Reply:
This is a perfectly constructed way to use it. I'm mostly thinking about if commutable should have more functions for this including createCodeCelll being exported.